### ISSUE TITLE
Breaking change / depcheck 

### ISSUE THREAD

**Issue by jessbowers** [dylang/npm-check#380]
The new version is released as a bug-fix (5.9.1) but contains a major change to depcheck. This causes downstream breakage when npm-check is used as a dependency inside a project. Specifically, node requirement has been updated to node 10+

Consider treating this as a major version bump (6.0)

**Comment by jessbowers** [comment]
The specific error we're seeing, when running in older node is regarding the `async` support:

```
/node_modules/depcheck/dist/check.js:70
async function getDependencies(dir, filename, deps, parser, detectors) {
      ^^^^^^^^
SyntaxError: Unexpected token function
    at Object.exports.runInThisContext (vm.js:78:16)
```

**Comment by LinusU** [comment]
Really sorry about this! Just released 5.9.2 which downgrades depcheck again, could you please test that version? Thanks!

**Comment by jessbowers** [comment]
Much better, thanks!
